features/support/cucumber_rails_helper.rb

```diff
@@ -2,38 +2,26 @@
 
 module CucumberRailsHelper
   def rails_new(options = {})
-    options[:name] ||= 'test_app'
-    command_result =
-      run_command "bundle exec rails new #{options[:name]} --skip-bundle --skip-test-unit --skip-spring --skip-bootsnap #{options[:args]}"
-    expect(command_result).to have_output(/README/)
-    expect(last_command_started).to be_successfully_executed
-    cd options[:name]
-    delete_environment_variable 'RUBYOPT'
-    delete_environment_variable 'BUNDLE_BIN_PATH'
-    delete_environment_variable 'BUNDLE_GEMFILE'
+    validate_rails_new_success(run_rails_new_command(options))
```

Comment:
I actually meant merging the actual two methods :smile:

Reply:
Ah whoops. I had to uncouple them to reduce the ABC score. This method was around 19, and I think now it scores around 10 or 11. If I put them back together, the score goes back over 15 :(

We could maybe alter the metric. But I'm not sure we should?

Reply:
> If I put them back together, the score goes back over 15 :(

:cry: 

Let's leave like this then :man_shrugging:.